src/pages/illustration/isometric-style/production.mdx

```diff
@@ -52,7 +52,7 @@ objects and create perspective.
 
 ### Automated object transformations
 
-1. Download the [isometric style starter kit]()
+1. Download the [isometric style starter kit]()(coming soon).
```

Comment:
I'm worried about having a link that doesn't link out anywhere. It feels misleading and maybe confusing for people, and Idk what it's entirely accessible either. Maybe we should just remove the link for now just say "coming soon" in italics afterward or something.

1. Download the isometric style starter kit _(coming soon)_.

Reply:
Good call @jnm2377 we can remove the link for sure. We should have this link ready in the next few weeks so it's for a very short period but fully agree that the link should not be active like it has been.